python/3d/auto_generated/qgs3dmapsettings.sip.in

```diff
@@ -348,6 +348,20 @@ Sets whether to show camera's view center as a sphere (for debugging)
 Returns whether to show camera's view center as a sphere (for debugging)
 
 .. versionadded:: 3.4
+%End
+
+    void setShowCameraRotationCenter( bool enabled );
```

Comment:
This kind of thing is better suited to a 

     void setFlags( Qgis::MapSettings3DFlags flags );

type approach, where Qgis::MapSettings3DFlags is a enum class in the Qgis namespace. It's more easily extensible and avoids the need for multiple getters/setters for boolean attributes like this.

Reply:
I see that this was newly intoduced code style, but I can't see how exactly these settings are being saved to the settings files ? Or is it some macros magic

Reply:
You can use qgsFlagValueToKeys to convert the flags to a string, and qgsFlagKeysToValue to convert back from string. See eg. MarkerLinePlacements in qgis.h line 1060, and then qgslinesymbollayer.cpp lines 1627 and 1502